regression/goto-instrument/restrict-function-pointer-to-multiple-functions-via-file-and-command-line-options/test.desc

```diff
@@ -0,0 +1,6 @@
+CORE
+test.c
+--function-pointer-restrictions-file restrictions.json --restrict-function-pointer use_f.function_pointer_call.1/g
```

Comment:
:question: Wouldn't this test work if it was ignoring the command line arguments too?

Reply:
This is testing the error message we get from this pass, if the command line arguments were ignored you’d get a different error message or none at all.